docs/README.md

```diff
@@ -191,6 +191,8 @@ options are available:
 |`LOGDNA_INCLUSION_REGEX_RULES`<br>**Deprecated**: `LOGDNA_INCLUDE_REGEX`|Comma separated list of regex patterns to include files from monitoring||
 |`LOGDNA_LINE_EXCLUSION_REGEX`|Comma separated list of regex patterns to exclude log lines. When set, the Agent will NOT send log lines that match any of these patterns.||
 |`LOGDNA_LINE_INCLUSION_REGEX`|Comma separated list of regex patterns to include log lines. When set, the Agent will send ONLY log lines that match any of these patterns.||
+|`LOGDNA_K8S_METADATA_EXCLUSION`|Comma separated list of Kubernetes selectors to exclude log lines associated with various Kubernetes resources||
+|`LOGDNA_K8S_METADATA_INCLUSION`|Comma separated list of Kubernetes selectors to include log lines associated with various Kubernetes resources||
```

Comment:
order of processing of these?

Reply:
line exclusion/inclusion rules have LINE in name - like LOGDNA_LINE_. LOGDNA_K8S_METADATA_EXCLUSION may imply  exclusion of the metadata itself, which is not the case here.

Reply:
Also, what is the process we are using MZ_ as a prefix instead of LOGDNA_ ? Just a heads up.